src/link.zig

```diff
@@ -1818,6 +1818,8 @@ pub const Input = union(enum) {
         needed: bool,
         weak: bool,
         reexport: bool,
+        name: ?[]const u8,
+        lib_directory: Directory,
```

Comment:
These new fields need to be handled in `hashInputs`.

Reply:
I've added these to `hashInputs`